api/lib/application/schooling-registration-dependent-users/schooling-registration-dependent-user-controller.js

```diff
@@ -23,6 +24,28 @@ module.exports = {
     return h.response(userSerializer.serialize(createdUser)).created();
   },
 
+  async createAndReconcileUserToSchoolingRegistrationByToken(request, h) {
+    const {
+      'campaign-code': campaignCode,
+      'external-user-token': token,
+      birthdate } = request.payload.data.attributes;
+
+    const createdUser = await usecases.createAndReconcileUserToSchoolingRegistrationByToken({ campaignCode, token, birthdate });
+
+    const accessToken = tokenService.createTokenFromUser(createdUser, 'external');
```

Comment:
Access Token = identifier + autoriser
```suggestion
    const accessToken = tokenService.createAccessTokenFromUser(createdUser, 'external');
```

Reply:
Done